debian/changelog

```diff
@@ -0,0 +1,5 @@
+mu (0.1-1) unstable; urgency=low
```

Comment:
Looks like we'll have to synchronise this with https://github.com/ntoll/mu/blob/master/mu/__init__.py. Looking at https://github.com/ntoll/mu/issues/58 I guess mu is currently being installed as 0.1-1 as listed here.
I'll see if I can have a look tonight at the link you posted.